R/linters_config.R

```diff
@@ -1,24 +1,22 @@
+gDR_undesirable_functions <-
+  lintr::modify_defaults(
+    defaults = lintr::default_undesirable_functions,
+    "library" = NULL,
```

Comment:
I think the list of exceptions will be longer (e.g. `Sys.setenv, options and mapply`).
Please create a dedicated branch in gDRgenesis and link gDRstyle GDR-1975 to it. Then enable all tests for all packages in ci/packages.yaml (with `test: y` tag). This step will enable gathering the lintr data for all the packages.

Reply:
I have a plan to merge gDRstyle at the end. So, first test it locally on all packages in scope of GDR-1975.
After that I ll check as you proposed.